### PR TITLE
 app-text/master-pdf-editor: version bump to 4.3.82

### PR DESCRIPTION
**Pull request by jpataias**

### CODE REVIEW

**Comment by jpataias** [comment]
I thought I had removed all the *.orig files.

How can I perform the rebase and commit removal know?

**Comment by jpataias** [comment]
And I actually don't understand how the commits from my previous pull request appear as new now.

**Comment by mgorny** [comment]
I'm sorry but I can't afford to give you lessons on using git. Please take some initiative, there's a lot of good guides on the subject online. https://git-scm.com/book/ being the official guide.

**Comment by jpataias** [comment]
Sorry about that, I just wasn't sure how to do it and how would it affect the pull request.

I believe everything is done now.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils eutils versionator
+
+DESCRIPTION="Master PDF Editor is a complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+
```
Comment (mgorny) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild]: Please strip this empty line, we group HOMEPAGE and SRC_URI together.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	x11-libs/libXext
+	>=dev-qt/qtsvg-5.4:5
+	>=dev-qt/qtnetwork-5.4:5
+	>=dev-qt/qtgui-5.4:5
+	>=dev-qt/qtprintsupport-5.4:5
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+S="${WORKDIR}/${PN}-$(get_major_version ${PV})"
```
Comment (mgorny) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild]: Please don't use versionator. Either use eapi7-ver or trivial bash pattern `${PV%%.*}`.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+src_install() {
+	local dest=/opt/${PN}
+	local bin_name=masterpdfeditor4
```
Comment (mgorny) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild]: Please don't introduce unnecessary temporary variables, they cause readability to suffer for no apparent gain.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+	insinto ${dest}
+	doins -r fonts lang stamps templates ${bin_name}.png
+
+	exeinto ${dest}
+	doexe ${bin_name}
+
+	dosym ${dest}/${bin_name} /opt/bin/${bin_name}
```
Comment (mgorny) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild]: Don't do absolute symlinks. Make it relative instead (with appropriate `../`).

File: app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+	make_desktop_entry ${bin_name} \
```
Comment (mgorny) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild]: This is desktop.eclass now, so replace eutils inherit.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+		"Master PDF Editor ${PV}" ${dest}/${bin_name}.png \
+		"Office;Graphics;Viewer" \
+		"MimeType=application/pdf;application/x-bzpdf;application/x-gzpdf;\nTerminal=false"
+}
+
+pkg_postinst() {
+	sed -i -e \
```
Comment (mgorny) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.82.ebuild]: This does not belong in postinst. Modify the file before installing it, in `src_prepare`.

**Comment by jpataias** [comment]
Just added the detailed changes to the commit.

**Comment by mgorny** [comment]
Thanks. I'm going to test it, and merge if it builds fine.

**Comment by mgorny** [comment]
Good enough for now. I have two comments for the future:

1.
       * QA Notice: Files built without respecting CFLAGS have been detected
       *  Please include the following list of files in your report:
       * /opt/master-pdf-editor/masterpdfeditor4
       * QA Notice: Pre-stripped files found:
       * /opt/master-pdf-editor/masterpdfeditor4
    Fix that via QA_PREBUILT (`man 5 ebuild`).
2.     >>> /usr/share/applications/masterpdfeditor4_%f-master-pdf-editor.desktop
    I don't think that's a good path.

**Comment by jpataias** [comment]
I didn't noticed that path problem, how can I add %f to the make_desktop_entry?

I will take these into consideration in the future.

Thanks for your work.

**Comment by mgorny** [comment]
I'm sorry but I have no clue. Please look at the code or ask around.

**Pull request closed without merging**